cliquet/resource/__init__.py

```diff
@@ -611,6 +611,11 @@ def _add_cache_header(self, response):
         collection_expires = self.request.registry.settings.get(setting_key)
         if collection_expires is not None:
             response.cache_expires(seconds=int(collection_expires))
+        else:
+            # Since Expires has resolution in seconds, do not use cache_expires
+            # in order to omit it.
```

Comment:
Might be useful to add a comment about "`no-cache` directive does not mean don't cache (despite its name)", since this is so confusing.